src/main/java/org/reactome/release/cosmicupdate/COSMICUpdateUtil.java

```diff
@@ -0,0 +1,362 @@
+	{
+		// ...no-op
+	}
+	
+	static
+	{
+		DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd_kkmmss");
+		COSMICUpdateUtil.dateSuffix = formatter.format(LocalDateTime.now());
+	}
+	
+	/**
+	 * Validate the identifiers in the database by comparing them to the identifiers in the file.
+	 * @param updaters A map of updaters, keyed by COSMIC identifier.
+	 * @param COSMICFusionExportFile The path to the COSMIC Fusion Export file.
+	 * @param COSMICMutationTrackingFile The path to the COSMIC Mutation Tracking file.
+	 * @param COSMICMutantExportFile The path to the COMSIC Mutant Export file.
+	 * @throws IOException
+	 * @throws FileNotFoundException
+	 */
+	static void validateIdentifiersAgainstFiles(Map<String, List<COSMICIdentifierUpdater>> updaters, String COSMICFusionExportFile, String COSMICMutationTrackingFile, String COSMICMutantExportFile) throws IOException, FileNotFoundException
+	{
+		validateAgainstCosmicFusionExport(updaters, COSMICFusionExportFile);
+		validateAgainstCosmicMutationTracking(updaters, COSMICMutationTrackingFile);
+		validateAgainstCosmicMutantExport(updaters, COSMICMutantExportFile);
+	}
+	
+	private static void validateAgainstCosmicMutantExport(Map<String, List<COSMICIdentifierUpdater>> updaters, String COSMICMutantExportFile) throws IOException, FileNotFoundException
+	{
+		logger.info("Now checking with CosmicMutantExport.tsv...");
+		try(CSVParser parser = new CSVParser(new FileReader(COSMICMutantExportFile), CSVFormat.DEFAULT.withFirstRecordAsHeader().withDelimiter('\t')); )
+		{
+			parser.forEach(record -> {
+				String legacyID = record.get(COSMIC_LEGACY_MUTATION_ID);
+				String mutationID = record.get(COSMIC_MUTATION_ID);
+				String genomicID = record.get(COSMIC_GENOMIC_MUTATION_ID);
+				if (updaters.containsKey(legacyID))
+				{
+					updaters.get(legacyID).forEach(updater -> {
+						updater.setValid(true); // only VALID if in MutantExport...
+						updater.getMutationIDs().add(mutationID);
+						updater.setCosvIdentifier(genomicID);
+					});
+
+				}
+			});
+		}
+	}
+
+	private static void validateAgainstCosmicMutationTracking(Map<String, List<COSMICIdentifierUpdater>> updaters, String COSMICMutationTrackingFile) throws IOException, FileNotFoundException
+	{
+		logger.info("Now checking with CosmicMutationTracking.tsv...");
+		// Now we need to look through the HUGE file from COSMIC and see if we can map the identifiers...
+		try(CSVParser parser = new CSVParser(new FileReader(COSMICMutationTrackingFile), CSVFormat.DEFAULT.withFirstRecordAsHeader().withDelimiter('\t'));)
+		{
+			parser.forEach(record -> {
+				String legacyID = record.get(COSMIC_LEGACY_MUTATION_ID);
+				String mutationID = record.get(COSMIC_MUTATION_ID);
+				String genomicID = record.get(COSMIC_GENOMIC_MUTATION_ID);
+				if (updaters.containsKey(legacyID))
+				{
+					updaters.get(legacyID).forEach(updater -> {
+						updater.getMutationIDs().add(mutationID);
+						updater.setCosvIdentifier(genomicID);
+					});
```

Comment:
I noticed in the other 2 validation methods, `updater.setValid(true)` is included.  Is there something different about this validation that it's not needed over here?